File: src/co/soundwave/repository/genre/GenreSQL.kt
package co.soundwave.repository.genre

import co.soundwave.repository.BaseSQL

class GenreSQL : BaseSQL() {
    fun insertGenres(name: String, path: String): String =
        "insert into ${tableName()}(name_${tableName()}, img_path_${tableName()}) values ('$name', '$path');"
    
    override fun tableName(): String = "genre"
}

File: repository/genre/GenreSQL.py
from repository import BaseSQL as bq


class GenreSQL(bq.BaseSQL):

    def insert_genre(self,_id,_name,_img_path_genre):
        return "insert into %s values (%s,%s,%s);" % (self.table_name(), _id, _name, _name)

    def table_name(self):
        return "genre"

    def abc(self):
        pass
